plugins/octant/cmd/antrea-octant-plugin/antrea_info.go

```diff
@@ -39,29 +39,28 @@ const (
 	heartbeatCol      = "Last Heartbeat Time"
 )
 
-func controllerHandler(request service.Request) (component.ContentResponse, error) {
+func (a *antreaOctantPlugin) controllerHandler(request service.Request) (component.ContentResponse, error) {
 	return component.ContentResponse{
 		Title: component.TitleFromString(controllerTitle),
 		Components: []component.Component{
-			getControllerTable(request),
+			a.getControllerTable(request),
 		},
 	}, nil
 }
 
-func agentHandler(request service.Request) (component.ContentResponse, error) {
+func (a *antreaOctantPlugin) agentHandler(request service.Request) (component.ContentResponse, error) {
 	return component.ContentResponse{
 		Title: component.TitleFromString(agentTitle),
 		Components: []component.Component{
-			getAgentTable(request),
+			a.getAgentTable(request),
 		},
 	}, nil
 }
 
 // getControllerTable gets the table for displaying Controller information
-func getControllerTable(request service.Request) *component.Table {
-	controllers, err := client.ClusterinformationV1beta1().AntreaControllerInfos().List(context.TODO(), v1.ListOptions{
-		ResourceVersion: "0",
-	})
+
+func (a *antreaOctantPlugin) getControllerTable(request service.Request) *component.Table {
+	controllers, err := a.client.ClusterinformationV1beta1().AntreaControllerInfos().List(context.TODO(), v1.ListOptions{})
```

Comment:
The resourceVersion is added by purpose, don't revert it, check the comment of this field for its meaning.

Reply:
Field added.